std/traits.d

```diff
@@ -7721,13 +7732,13 @@ template getSymbolsByUDA(alias symbol, alias attribute)
     static assert(hasUDA!(getSymbolsByUDA!(HasPrivateMembers, Attr)[0], Attr));
 }
 
-///
+//
 @safe unittest
```

Comment:
Access to private members is __not__ possible with `getSymbolsByUDA` - the test only works it's in the same module.
[Convince yourself](https://is.gd/lrpAqB)

Reply:
I suggest moving out the bad parts of the unit test rather than hiding the entire test.